font-codegen/src/parsing.rs

```diff
@@ -598,8 +610,42 @@ impl Parse for FieldType {
     }
 }
 
+fn is_wellknown_scalar(path: &syn::PathSegment) -> bool {
+    if !path.arguments.is_empty() {
+        return false;
+    }
+    // TODO use spec types not Rust types
+    // TODO feels like this should be an enum, know it's size, etc
```

Comment:
agree, we should have an enum that we parse directly.

Reply:
Done